source/read-from-mongodb.txt

```diff
@@ -42,6 +42,18 @@ Overview
 
          .. include:: /scala/filters.txt
 
+.. important:: Change Stream Schema Inference
+
+   When the {+driver-short+} infers the schema of a data frame 
+   read from a change stream, by default,
+   it will use the schema of the underlying collection rather than the schema
+   of the change stream. To instruct the connector to use the schema of the 
```

Comment:
```suggestion
   When the {+driver-short+} infers the schema of a data frame 
   read from a change stream, by default,
   it will use the schema of the underlying collection rather than of the change stream.
   To instruct the connector to use the schema of the 
```

Reply:
Used your suggestion plus "that" to hopefully make it more clear. LMK